nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -102,16 +102,37 @@ fi
 extraBuildFlags+=(--option "build-users-group" "$buildUsersGroup")
 
 # Inherit binary caches from the host
-# TODO: will this still work with Nix 1.12 now that it has no perl? Probably not... 
+# TODO: will this still work with Nix 1.12 now that it has no perl? Probably not...
 binary_caches="$(@perl@/bin/perl -I @nix@/lib/perl5/site_perl/*/* -e 'use Nix::Config; Nix::Config::readConfig; print $Nix::Config::config{"binary-caches"};')"
 extraBuildFlags+=(--option "binary-caches" "$binary_caches")
 
 nixpkgs="$(readlink -f "$(nix-instantiate --find-file nixpkgs)")"
 export NIX_PATH="nixpkgs=$nixpkgs:nixos-config=$mountPoint/$NIXOS_CONFIG"
```

Comment:
Not sure if it matters much but this would propagate "configured-for-installation" NIX_PATH into the chroot environment.

Reply:
I _think_ that's what it used to do. In case you're missing context, I basically rewrote how `nixos-install` worked a couple of months ago, and forgot to support the `--chroot` option at the time. This is me just bringing it back to life, not trying to do anything fancy.